src/OidcClient.php

```diff
@@ -436,7 +478,7 @@ private function requestTokens(
     // Clear code verifier from session after check
     $this->sessionStorage->clearCodeVerifier();
 
-    return new OidcTokens($jsonToken);
+    return $jsonToken;
```

Comment:
Now we can return the `UnvalidatedOidcTokens` here

Reply:
:heavy_check_mark: